### ISSUE TITLE
inferno-tools

### ISSUE THREAD

**Issue by dbismut** [infernojs/create-inferno-app#29]
Hey - just a question : considering that `inferno-tools` is needed to benefit from React chrome extension, wouldn't it make sense to ship it with `create-inferno-app` and requiring it when `process.env.NODE_ENV` is set to `development`?

**Comment by Havunen** [comment]
Hi @dbismut  thanks for suggestion. We could do this, why not? Can you send PR for it?

**Comment by dbismut** [comment]
Of course I can try, but I'm no pro developer and not sure where to start. Is looking at the `webpack.config.dev.js` a good idea? Can I require and execute `inferno-devtools` from there? Thanks for your help ;)

**Comment by dbismut** [comment]
Hi @Havunen, after trying further, I tried to modify the entry from `webpack.config.dev.js` with the following:

```javascript
modules.export = {
//...
  entry: [
    //...
    require.resolve('inferno-devtools');
    paths.appIndexJs
  ]
//...
}
```

I know that `inferno-devtools` are indeed executed, but for some reason the React extension on Chrome doesn't load. What's weird is that if I copy the distilled code of the `inferno-devtools` nom package into a `devtools.js` file that I put next to the appIndexJs file, and then requiring that file in the entry point instead of the npm package, everything runs fine. Not sure what happens there 🙃

**Comment by Havunen** [comment]
Yeah, I think devtools must be imported from index.js (entry point) before other stuff.

**Comment by dbismut** [comment]
Well I don't understand why this works 🤕

```javascript
modules.export = {
//...
  entry: [
    //...
   '/Users/david/repos/create-inferno-app/packages/inferno-scripts/template/src/devtools.js',
    paths.appIndexJs
  ]
//...
}
```

and this doesn't:

```javascript
modules.export = {
//...
  entry: [
    //...
   '/Users/david/repos/create-inferno-app/node_modules/inferno-devtools/dist/inferno-devtools.js',
    paths.appIndexJs
  ]
//...
}
```

**Comment by Havunen** [comment]
Ping @longlho hes good with this stuff.

**Comment by farooqkz** [comment]
As `inferno-devtools` has been retired, I think this issue is not valid anymore?
